src/daal4py.cpp

```diff
@@ -871,12 +868,7 @@ double c_roc_auc_score(data_or_file & y_true, data_or_file & y_test)
     auto table_true = get_table(y_true);
     auto table_test = get_table(y_test);
     auto type       = (*table_test->getDictionary())[0].indexType;
-    if (type == daal::data_management::data_feature_utils::DAAL_FLOAT64 ||
-        type == daal::data_management::data_feature_utils::DAAL_INT64_S ||
-        type == daal::data_management::data_feature_utils::DAAL_INT64_U ||
-        type == daal::data_management::data_feature_utils::DAAL_FLOAT32 ||
-        type == daal::data_management::data_feature_utils::DAAL_INT32_S ||
-        type == daal::data_management::data_feature_utils::DAAL_INT32_U)
+    if (type == daal::data_management::data_feature_utils::DAAL_FLOAT64 || type == daal::data_management::data_feature_utils::DAAL_INT64_S || type == daal::data_management::data_feature_utils::DAAL_INT64_U || type == daal::data_management::data_feature_utils::DAAL_FLOAT32 || type == daal::data_management::data_feature_utils::DAAL_INT32_S || type == daal::data_management::data_feature_utils::DAAL_INT32_U)
```

Comment:
Is it right autoformatting?

Reply:
clang-format fixed it, I'll get it back. Thanks